NHS.Api/Python3.Riak.Ubuntu/NHS_AnalyticsAPI/views.py

```diff
@@ -30,7 +32,8 @@ def search_trust(request, trust_name):
         "Region_Data": region_dict,
         "Trust_Data": peer_list,
     }
-    return JsonResponse(resp_data, status=200)
+    resp_data = json.dumps(resp_data, cls=DjangoJSONEncoder)
+    return JsonResponse(resp_data, safe=False, status=200)
```

Comment:
- remove  status=200, check ,
- why we want "safe=False"
- as we are not doing anything on object "resp_data" we can replace it by directly using statement